docs/UserGuide.md

```diff
@@ -437,18 +445,21 @@ Example:
 <br>
 
 <a name="find"></a>
+
 #### Finding an applicant by name: `find`
 
-Find employees whose name contains a particular keyword.
+Find applicants whose name contains a particular keyword.
 
 Format: `find KEYWORD [MORE_KEYWORDS]`
+
 * The search is case-insensitive, e.g. `JOHN` will return `john`.
 * The order of the keywords does not matter, e.g. `Alice Tan` will match `Tan Alice`.
 * Only the applicant name is searched.
 * Any applicant whose name contains the sequence of characters given as the keyword will be given as a result, e.g. `Ed` will match both `Edward` and `Ed`.
-* Applicants matching at least one keyword will be returned (i.e. OR search), e.g. `Ben Bobby` will return `Ben Yang`, `Bobby Chin`.
+* Applicants matching at least one keyword will be returned (i.e. OR search), e.g. `Ben Bobby` will return `Ben Yang` and `Bobby Chin`.
```

Comment:
Perhaps can include the constraint that the total length of keyword(s) input must be at most 55 characters?